### ISSUE TITLE
Contact information

### ISSUE THREAD

**Issue by vittorioromeo** [MaikKlein/maikklein.github.io#4]
Hello Maik, I am interested in entity component systems and I have created a compile-time experiment like the one from your excellent article. I'm also currently working on an updated multithreading-friendly version of that.

I wanted to contact you to ask you some advice regarding the design of some parts of the library _(mostly regarding the addition/deletion of components)_ but found no email address or contact information, and it doesn't seem like your blog supports comments. 

If this is intentional, no problem at all - just thought I'd mention it to you in case it was an oversight.

**Comment by MaikKlein** [comment]
Yes I am currently switching from Jekyll to Hugo and I haven't updated everything yet. You can post any questions here in the issues and I will try to answer them as best as I can.
